initial_position: [0.0, 0.0, deg(90)]
goal_position: [8.0, 2.0]
intermediate_points:
  - [2.0, 3.0]
  # - [6.0, 3.0]
  # - [10.0, 2.0]
  # - [10.0,-1.0]
  # - [6.0, -2.0]
distance: 1.5
ang_param:
  kp: 1.5
  ki: 0.0
  kd: 0.0
lin_param:
  kp: 1.0
  ki: 0.0
  kd: 0.0